package.json

```diff
@@ -1,4 +1,94 @@
 {
   "name": "builder-react-component",
-  "version": "0.0.0"
+  "version": "0.0.1",
+  "description": "Builder Archetype - React Component",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/FormidableLabs/builder-react-component.git"
+  },
+  "author": "Ryan Roemer <ryan.roemer@formidablelabs.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/FormidableLabs/builder-react-component/issues"
+  },
+  "homepage": "https://github.com/FormidableLabs/builder-react-component",
+  "scripts": {
+    "npm:postinstall": "node -e \"require('fs').stat('lib', function(e,s){process.exit(e || !s.isDirectory() ? 1 : 0)})\" || builder run build-lib",
+    "npm:preversion": "builder run check",
+    "npm:version": "builder run clean && builder run build && git add -A dist",
+    "npm:test": "builder run test-frontend",
+    "clean-dist": "rimraf dist",
+    "build-dist-min": "webpack --config node_modules/builder-react-component/config/webpack/webpack.config.js",
```

Comment:
^ i.e., Builder views your repo structure as "layered over" the archetype's, and any paths that don't *explicitly* point to the archetype's in node_modules, should pick up your file (if it exists), or the archetype's (if not).